config/schema/document_types/policy.json

```diff
@@ -1,5 +1,7 @@
 {
   "fields": [
-    "slug"
+    "slug",
+    "organisations",
```

Comment:
OK, in that case I'll close this PR since I've just discovered people aren't yet taggable to new policies.  We've another ticket for handling people on policies so this can be handled then, since it isn't a "while I'm here" task anymore.